### DISCUSSION TITLE
Can we drill down "Others" categories?

### DISCUSSION THREAD

**Discussion by ajmal-yazdani** [mivano/azure-cost-cli#12] [Ideas]
It's very excellent to see all the resource cost by service name where I am seeing lots of services cost going to "Other" head and showing as a cumulative cost .

Can we drill down all actual services and shows all services cost instead of "Other":

![image](https://user-images.githubusercontent.com/1866370/232488728-63adb143-70b9-4d85-9345-826f8d0950cc.png)

**Comment by mivano** [comment]
We certainly can as the data is already available, however, I m trying to trim it down so it fits on the screen. What I can do is make it at least configurable.

In the Json output, you get all the data BTW.

**Comment by mivano** [comment]
This is fixed as per issue #13

You can now control when it will rollup the remaining categories in a Others category.
